cg/services/orders/store_order_services/store_fastq_order_service.py

```diff
@@ -126,11 +132,28 @@ def store_items_in_status(
                 ):
                     self.create_maf_case(sample_obj=new_sample)
                 case.customer = customer
-                new_relationship = self.status.relate_sample(
+                new_relationship = self.status_db.relate_sample(
                     case=case, sample=new_sample, status=StatusEnum.unknown
                 )
-                self.status.session.add_all([case, new_relationship])
+                self.status_db.session.add_all([case, new_relationship])
 
-        self.status.session.add_all(new_samples)
-        self.status.session.commit()
+        self.status_db.session.add_all(new_samples)
+        self.status_db.session.commit()
         return new_samples
+
+    @staticmethod
+    def _fill_in_sample_ids(samples: list[dict], lims_map: dict, id_key: str = "internal_id"):
```

Comment:
Was there a reason for moving this from the parent class? Was it to add clarity?

Reply:
The overall idea is explained here:
https://github.com/Clinical-Genomics/cg/pull/3574#discussion_r1719593949

Ideally the store service will be the same for all later on with different data transfer services as required. Right now this implementation is not there as the code needs to be further disentangled.

Reply:
But it seems like we have a method, `_fill_in_sample_ids` which is identical in all child classes. What prevents it from being moved to `StoreOrderService`?

Reply:
moved it, Wanted to keep the ABC classes clean with the future refactor of the store service in mind which would hopefully reduce it to a single service. But we can tackle that later on.